helmfile/charts/prometheus-alerts/templates/rules/daily-checks.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.dailyChecks}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "daily-checks" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: daily-checks
```

Comment:
will you also create alerts for alasticsearch snapshot retention?

Reply:
Hmm, I think it's halfway covered in the backup status dashboard right? anything in particular about it you would like to monitor?

Reply:
maybe just to count the number of snapshots and trigger an alert if the number suddenly increases.
this could indicate a retention job issue

Reply:
This kind of "increase" check has turned out to be very hard to do for now as influxdb doesn't store enough metrics over a long enough time to make it reliable. We would need metrics of about a month rather than just 1 week. This can be investigated further as the new metrics database, "thanos", is in place.